03-account-kit-manage-resources/index.ts

```diff
@@ -0,0 +1,37 @@
+// SDK exposes all you need
+import { Chains } from '@[SOMETHING]kit'
+import { Asset, RamBytes, Account, SigningRequest } from '@[SOMETHING]kit/account'
+
+async function main() {
+    // Loads account
+    const account: Account = await Account.load("teamgreymass", Chains.EOS)
+
+    // Buy ram
+    const buyRamEsr: SigningRequest = account.resources.ram.buy(RamBytes.from(1000))
+
+    // You could then pass it to Session Kit to sign and broadcast
+    // session.handle(buyRamEsr)
+
+    // Sell ram
+    const sellRamEsr: SigningRequest = account.resources.ram.sell(RamBytes.from(1000))
+
+    // Delegate bandwidth
+    const delegatebwEsr: SigningRequest = account.resources.net.stake(Asset.from(1, "EOS"))
+
+    // Undelegate bandwidth
+    const undelegatebwEsr: SigningRequest = account.resources.net.unstake(Asset.from(1, "EOS"))
+
+    // Delegate CPU
+    const delegateCPUEsr: SigningRequest = account.resources.cpu.stake(Asset.from(1, "EOS"))
+
+    // Undelegate CPU
+    const undelegateCpuEsr: SigningRequest = account.resources.cpu.unstake(Asset.from(1, "EOS"))
+
+    // Powerup
+    const powerupEsr: SigningRequest = account.resources.powerup(Asset.from(1, "EOS"))
```

Comment:
The `powerup` method may want to use milliseconds as the input, and do all the calculations to determine the amount. There's actually no amount in powerup, besides the max the user is willing to pay. 

As reference, here's the PowerUp class from the eosio-resources lib:

https://github.com/greymass/eosio-resources/blob/master/src/powerup/cpu.ts

Reply:
Ahh good to know, thanks!